dvl-a50/mavlink2resthelper.py

```diff
@@ -352,12 +352,12 @@ def send_vision_position_estimate(
         )
         logger.info(post(MAVLINK2REST_URL + "/mavlink", data=data))
 
-    def send_rangefinder(self, distance: float):
+    def send_rangefinder(self, distance: float, quality: float):
         "Sends message DISTANCE_SENSOR to flight controller"
         if distance == -1:
-            return
-        data = self.rangefinder_template.format(int(distance * 100))
-
+            data = self.rangefinder_template.format(int(0), 0)
```

Comment:
Is the `int` here just for consistency with the `else` case? Seems redundant to cast an integer literal to an integer, but not super fussed either way (although it's a bit weird to only be done to one of them).

Reply:
nah, my mistake, I had a multiplication there before, I removed it and this got left over